prettier.go

```diff
@@ -0,0 +1,170 @@
+package metricsql
+
+import (
+	"fmt"
+	"strings"
+)
+
+const indentString = "  "
+
+func paddings(indent int) string {
+	var str strings.Builder
+	for i := 0; i < indent; i++ {
+		str.WriteString(indentString)
+	}
+	return str.String()
```

Comment:
Can this be replaced with `strings.Repeat`?
If that's true it seems like this function can be replaced with direct calls to `strings.Repeat`

Reply:
uses `strings.Repeat` and updated function